pom.xml

```diff
@@ -61,6 +61,11 @@
                 <artifactId>jaeger-core</artifactId>
                 <version>${jaeger.version}</version>
             </dependency>
+            <dependency>
```

Comment:
You can just use `jaeger-client` instead of `jaeger-core` and it will bring `jaeger-thrift` transitively.

Reply:
@jpkrohling ok, I will update this